style_guides/pl_sql_packages.md

```diff
@@ -0,0 +1,119 @@
+# Database Packages
+
+## Package Names
+
+The following naming conventions should be used for database packages:
+
+```
+pkw_<name>        -- Package
+pkw_test_<name>   -- UT/PLSQL Test package
+```
+
+## Package Functions and Procedures

```

Comment:
Pedant change, would change this to "Package Procedures and Functions" to be the same order as the content of the section